Makefile

```diff
@@ -60,7 +60,6 @@ build: deps fmt
 
 # Pushes the image to docker
 push: build
-	@docker push $(IMAGE_NAME)
```

Comment:
why this is removed??

Reply:
Remove `IMAGE_NAME` push from make command. IMO we should not push
    image with tag like `251ef80-dirty` for every commit. Instead latest will have updated
    changes and for every release we should modify the tag version and push images accordingly.